streams/writable-streams/aborting.js

```diff
@@ -131,6 +131,20 @@ promise_test(t => {
     'rejection reason of abortPromise must be the error thrown by abort');
 }, 'WritableStream if sink\'s abort throws, the promise returned by writer.abort() rejects');
 
+promise_test(t => {
+  const ws = new WritableStream({
+    abort() {
+      throw error1;
+    }
+  });
+  const writer = ws.getWriter();
+
+  return Promise.all([
+    promise_rejects(t, error1, writer.abort(undefined), 'first abort() must have matching rejection'),
+    promise_rejects(t, error1, writer.abort(undefined), 'second abort() must have matching rejection')
+  ]);
+}, 'WritableStream if sink\'s abort throws, the promise returned by multiple writer.abort()s reject');
```

Comment:
I think it would also be good to have a check that the Promises returned are actually the same object. In fact, combining it into this test would probably be okay.